dotcom-rendering/fixtures/manual/editionsCrossword.ts

```diff
@@ -1,8 +1,8 @@
-import type { FEEditionsCrossword } from '../../src/types/editionsCrossword';
+import type { CAPICrossword } from '@guardian/react-crossword/dist/@types/CAPI';
```

Comment:
Non-blocking: we should probably avoid importing from this directory. In the article crossword player we have something like this to get access to this type:

```tsx
import type { CrosswordProps } from '@guardian/react-crossword';

type CAPICrossword = CrosswordProps['data'];
```